hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/Utils.java

```diff
@@ -50,6 +50,17 @@ public static Optional<HederaKey> asHederaKey(final Key key) {
         }
     }
 
+    // This method shouldn't be here. It needs to find a new home.
+    public static Optional<HederaKey> asHederaKeyUnchecked(final Key key) {
```

Comment:
Why can't we use `asHederaKey` method below ?

Reply:
this is just for testing because some of the key generated by seeded source are not valid

Reply:
Got it. Thanks